libdnf/repo/DependencySplitter.cpp

```diff
@@ -29,7 +29,7 @@
 namespace libdnf {
 
 static const Regex RELDEP_REGEX = 
-    Regex("^(\\S*)\\s*(<=|>=|<|>|=|==)?\\s*(\\S*)$", REG_EXTENDED);
+    Regex("^([^[:space:]]*)[[:space:]]*(<=|>=|<|>|=|==)?[[:space:]]*([^[:space:]]*)$", REG_EXTENDED);
```

Comment:
Interesting. It's never come to my mind that there might be a compatibility problem with '\s'.
Does it make sense to leave a comment here? Just to remind future maintainers who might feel tempted to "simplify" the regex by putting `\s` back.

Would you mind filing a similar PR also for dnf5 (next generation of dnf)?
https://github.com/rpm-software-management/dnf5/blob/e2aa5eead7589e65994b2f971f63d5d64b88d421/libdnf5/solv/reldep_parser.cpp#L30

Reply:
> Does it make sense to leave a comment here?

Sure, I updated the diff.

> Would you mind filing a similar PR also for dnf5 (next generation of dnf)?

Certainly: https://github.com/rpm-software-management/dnf5/pull/843